src/strbuf.c

```diff
@@ -6,32 +6,32 @@
 #include "strbuf.h"
 
 struct Strbuf_t {
-    u16 maxsize;
+    u16 maxSize;
     u16 size;
-    u32 magic;
-    Charcode data[1];
+    u32 checksum;
```

Comment:
I think `integrity` is a better name since it's not actually calculated from a checksum function it's more like a check for if the string exists or not.